wire-ios/Wire-iOS/Sources/UserInterface/SelfProfile/ProfileHeaderViewController.swift

```diff
@@ -248,6 +249,8 @@ final class ProfileHeaderViewController: UIViewController {
             teamObserver = TeamChangeInfo.add(observer: self, for: team)
         }
         view.backgroundColor = .clear
+
+        qrCodeButton.addTarget(self, action: #selector(qrCodeButtonTapped), for: .touchUpInside)
     }
```

Comment:
We're trying to move away from the selector pattern. What we can use instead is a UIAction. 
```suggestion
        let qrCodeAction = UIAction { _ in
    self.qrCodeButtonTapped()
}
qrCodeButton.addAction(qrCodeAction, for: .touchUpInside)
    }
```

As a result, we can remove the @objc annotation from the `qrCodeButtonTapped` method.